.travis.yml

```diff
@@ -1,10 +1,8 @@
 sudo: false
 language: python
-python: '3.4'
+python: '3.10'
 env:
   matrix:
-  - TESTENV=py26
-  - TESTENV=py27
   - TESTENV=py34
```

Comment:
310?

Reply:
Added 3.9 and up (followed https://tox.wiki/en/latest/user_guide.html)